From: Colin Ian King <colin.i.king@gmail.com>
Subject: selftests/mm: fix spelling mistake "succedded" -> "succeeded"
Date: Tue, 21 Nov 2023 09:31:04 +0000

There is a spelling mistake in a ksft_exit_fail_msg message. Fix it.

Link: https://lkml.kernel.org/r/20231121093104.1728332-1-colin.i.king@gmail.com
Signed-off-by: Colin Ian King <colin.i.king@gmail.com>
Cc: Andrei Vagin <avagin@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/vm_util.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/testing/selftests/mm/vm_util.c~selftests-mm-check-that-pagemap_scan-returns-correct-categories-fix-2
+++ a/tools/testing/selftests/mm/vm_util.c
@@ -75,7 +75,7 @@ static bool pagemap_scan_supported(int f
 	/* Provide an invalid address in order to trigger EFAULT. */
 	ret = __pagemap_scan_get_categories(fd, start, (struct page_region *) ~0UL);
 	if (ret == 0)
-		ksft_exit_fail_msg("PAGEMAP_SCAN succedded unexpectedly\n");
+		ksft_exit_fail_msg("PAGEMAP_SCAN succeeded unexpectedly\n");
 
 	supported = errno == EFAULT;
 
_
